src/main/webapp/resources/js/components/samples/components/fastqc/FastQCCharts.jsx

```diff
@@ -0,0 +1,80 @@
+import { Col, Image, Row, Typography } from "antd";
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
+import styled from "styled-components";
+
+import { getFastQCImages } from "../../../../apis/files/sequence-files";
+import { grey4 } from "../../../../styles/colors";
+import { SPACE_MD } from "../../../../styles/spacing";
+import { InfoAlert } from "../../../alerts";
+
+const StyledImage = styled(Image)`
+  border: 1px solid ${grey4};
```

Comment:
change this to: `border: 1px solid var(--grey-4)` // Make sure the color is dark enough, there is a slight difference